display.c

```diff
@@ -162,6 +162,13 @@ void exitCurses(void)
   endwin();
 }
 
+static void printaddr(uint64_t addr)
+{
+  char templ[7];
```

Comment:
`7` needs to be explained.
Afaik it is `sizeof("%016lX")` with 16 being the max `nAddrDigits`